scripts/build-image.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+set -e
+CURL="curl -sSf"
+FLANNEL_CNI_ROOT=$(git rev-parse --show-toplevel)
+IMAGE_NAME=quay.io/coreos/flannel-cni
+VERSION=$($FLANNEL_CNI_ROOT/scripts/git-version)
+CNI_VERSION="v0.5.2"
```

Comment:
Hmmm - this does seem like the same process as in the makefile, no? Any reason we need to the makefile and the build-image script?